src/csvcubed/models/cube/qb/components/attributevalue.py

```diff
@@ -36,3 +51,23 @@ def get_permitted_rdf_fragment_hints(self) -> Set[RdfSerialisationHint]:
 
     def get_identifier(self) -> str:
         return self.label
+
+    def validate_data(
```

Comment:
This function doesn't need to be here. Make sure to tidy up the imports once you get rid of it.

Reply:
Removed